src/game.cpp

```diff
@@ -17,21 +17,25 @@ GameState *GAMESTATE() { return _global_gs; }
 #endif
 
 GFX::Mesh mesh;
-GFX::Shader shader;
 GFX::Texture texture;
+GFX::Camera camera;
 
 void init_game(GameState *gamestate) {
     _global_gs = gamestate;
     Asset::load("assets.bin");
 
-    GFX::init(GAMESTATE(), Asset::fetch_shader("MASTER_SHADER")->data, 600, 600);
+    GFX::init(GAMESTATE(), 600, 600);
 
     GAMESTATE()->running = true;
 
-    Input::bind(Ac::AButton, 0, SDLK_a, 1.0);
-    Input::bind(Ac::AButton, 1, SDLK_s, 0.1);
-    Input::bind(Ac::BButton, 0, SDLK_b, 1.0);
-    Input::bind(Ac::BButton, 1, SDLK_n, 0.1);
+    Input::bind(Ac::MoveX, 0, SDLK_a, -1.0);
+    Input::bind(Ac::MoveX, 1, SDLK_d,  1.0);
+    Input::bind(Ac::MoveZ, 0, SDLK_w,  1.0);
+    Input::bind(Ac::MoveZ, 1, SDLK_s, -1.0);
+    Input::bind(Ac::Jaw, 0, SDLK_q, -1.0);
+    Input::bind(Ac::Jaw, 1, SDLK_e,  1.0);
+    Input::bind(Ac::Pitch, 0, SDLK_i,  1.0);
+    Input::bind(Ac::Pitch, 1, SDLK_k, -1.0);
```

Comment:
Moving left/right and jaw feel inverted to me, is it the same for you? Might not matter that much, hopefully we'll have mouse movement.

Reply:
It should be inverted, since the camera moves the world no the camera. And it is inverted for me aswell... I honeslty just didn't bother with something that was going to be replaced with mouse controls. But I'm split on where the negation should be, if it should be in the camera or outside the camera. What do you think?